cla_public/templates/base.html

```diff
@@ -84,6 +118,36 @@
 
 {% block javascripts %}
   <script>
+
+    var currentLocation = window.location.href;
+    if (currentLocation.indexOf("n43n3") > 0 || currentLocation.indexOf("n43n14/n97") > 0 || currentLocation.indexOf("n88") > 0 || currentLocation.indexOf("n149") > 0 || currentLocation.indexOf("n86") > 0) {
+      $(".laa-flee-button-container").addClass("laa-fee-button-enabled");
+    }
+
+    function flee() {
+      $("body").hide();
+      {#
+        The below are all other possibilities but not necessarily the best approach. 
+
+        var histLength = history.length;
+        window.history.go(-histLength);
+        history.pushState({page: 1}, "Cookies", "/cookies")
+        history.pushState({page: 2}, "Bank Holidays in the UK", "/bank-holidays")
+      #}
+      window.open("http://bbc.co.uk/weather", "_newtab");
+      window.location.replace("http://www.google.co.uk");
+    }
+    $("#quick-exit").on("click", function() {
+      flee();
+      ga('moj-send', 'event', 'quick-exit', 'Mouse-click');
+    });
+    $(document).keyup(function(e) {
+      if (e.keyCode == 27) { // escape key
+        flee();
+        ga('moj-send', 'event', 'quick-exit', 'Escape-key');
```

Comment:
If flee is changing the browser location does the GA code still get executed?
Is it possible to rename flee to something more descriptive, i.e. quick_exit

Reply:
Hmmm... maybe best to move the location of `ga()` to before the `flee()`...